xml/favicons.xml

```diff
@@ -1810,7 +1810,7 @@
     <description>PBX in a Flash</description>
     <example>f5c62ea4c4e9f9a8606400becc01375e</example>
     <param pos="0" name="hw.vendor" value="PIAF"/>
-    <param pos="0" name="hw.device" value="PBX"/>
+    <param pos="0" name="hw.device" value="SIP Gateway"/>
```

Comment:
We've standardized on a few variants on `SIP` and `VoIP`. If PBX needs to be its own class we should do this across products.